executor/Cargo.toml

```diff
@@ -21,15 +21,17 @@ tokio.workspace = true
 tracing.workspace = true
 
 dashmap = "6.0"
-jsonrpsee = { version = "0.22", features = ["server"] }
 rust_decimal = "1.35"
 rust_decimal_macros = "1.35"
 serde_json = "1"
 clap = { version = "4.5.4", features = ["derive"] }
 hex-literal = "0.4.1"
+semver = "1.0.23"
+arc-swap = "1.7.1"
 
 [dev-dependencies]
 serial_test = "3.1.1"
+jsonrpsee-core = "0.22"
```

Comment:
is this only for async trait? can't we just import it directly

Reply:
don